WcaOnRails/app/views/static_pages/contact.html.erb

```diff
@@ -16,14 +16,15 @@
   <p>
     <strong><%= t('contact.board.name_html') %></strong> - <%= mail_to "board@worldcubeassociation.org" %><br />
     <%= t('contact.board.info_html') %><br />
-    <%= @board_members.order(:name).map(&:name).join(", ") %>
+    <%= Team.find_by_friendly_id(Team::BOARD_FRIENDLY_ID).current_members.includes(:user).map(&:user).sort_by(&:name).map(&:name).join(", ") %>
```

Comment:
This is failing at staging: https://hastebin.com/kuxayeqoru.xml

Works fine at the console and also locally. Not sure what's going on.